proto/stargaze/alloc/v1beta1/params.proto

```diff
@@ -5,6 +5,7 @@ package publicawesome.stargaze.alloc.v1beta1;
 option go_package = "github.com/public-awesome/stargaze/v11/x/alloc/types";
```

Comment:
Should we increase the proto version since the API is changing?

Reply:
the v1beta1 can stay since we are just adding new fields, usually in protobuf they come as optional, as for the v11 we already bumped it and is just essentially the go package name